omod/src/main/webapp/resources/scripts/app.js

```diff
@@ -0,0 +1,59 @@
+
+var app  = angular.module('Tag',['tagService','ngDialog','uicommons.common.error']);
+
+
+app.controller('tagCtrl',['$scope','TagService','ngDialog',
+    function ($scope,TagService,ngDialog) {
+
+       /**
+         * Method to load Tags
+         */
+        $scope.loadTags =function (personUuid) {
+            $scope.thisPersonUuid = personUuid;
```

Comment:
since the loadTags function requires a parameter, we need to pass the person uuid when we call it from the dialogs.  `personUuid` is a parameter passed from the view. i guess `thisPersonUuid` doesn't have to be a scope variable. It just needs to be a variable so that we can use `personUuid` outside of the loadTags function